examples/omni-lock-solana/README.md

```diff
@@ -0,0 +1,21 @@
+# Lumos Works with UniSat
+
+Nervos maintains a powerful lock
+called [Omnilock](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md) (previously named RC lock), which
+can use MetaMask as a signer. This example will show how to use Lumos to send a transaction using Omnilock and MetaMask
+
+## Quick Start
+
+> we should [build](..) Lumos project first before we start this example
+
+```
+npm run build
+cd examples/omni-lock-
+npm start
+```
+
+## Links
+
+- [MetaMask](https://unisat.io/) - A crypto wallet for Bitcoin
```

Comment:
The description here is wrong. You have to use sol and phantom. There may be similar errors in the description of the previous example.